lib/net/http_fetch_plugin.js

```diff
@@ -243,6 +243,16 @@ shaka.net.HttpFetchPlugin = class {
     } else {
       return false;
     }
+    // Old fetch implementations hasn't body and ReadableStream implementation
+    // See: https://github.com/shaka-project/shaka-player/issues/5088
+    if (window.Response) {
+      // eslint-disable-next-line no-restricted-syntax, no-prototype-builtins
+      if (!Response.prototype.hasOwnProperty('body')) {
```

Comment:
would `Object.prototype.hasOwnProperty.call(Response.prototype, 'body');` allow removal of the lint exceptions?

```suggestion
      if (!Object.prototype.hasOwnProperty.call(Response, 'body')) {
```